markdownPreview/index.ts

```diff
@@ -13,6 +14,7 @@ function init() {
             : lightModeTheme ?? 'default'
     };
     mermaid.initialize(config);
+    mermaid.registerExternalDiagrams([mindmap], {lazyLoad: false});
```

Comment:
Does `lazyLoad` need to be false? Would be nice to avoid the cost of initializing mindmap support if it's not needed

Reply:
I was trying to follow the presence of synchronous and asynchronous code. Whatever you want to do, I’m amenable.

Reply:
Try switching to lazy loading. That way we only pay the cost of loading this script when mind maps are actually needed

Reply:
done!